nx-X11/programs/Xserver/Xext/xres.c

```diff
@@ -25,7 +25,6 @@
 static int
 ProcXResQueryVersion (ClientPtr client)
 {
-    REQUEST(xXResQueryVersionReq);
```

Comment:
> "stuff" hardly seems like a good name for something to magically declare in a macro. "register" 
> also seems suspicious. But I guess that is for later.

```
#define REQUEST(type) \
        type *stuff = (type *)client->requestBuffer
```

The "stuff" variable name gets used quite often in X.org Xserver, actually. Above you find the REQUEST definition in X.org's Xserver. And the "register" is for later, indeed. Basically, all "register" attributes can be removed IMHO.